components/kubeflow-pipeline/pipeline_components/analytics-client/flwr_analytics_client/__main__.py

```diff
@@ -0,0 +1,22 @@
+from typing import List
+
+import flwr
+
+import data
+from client import AnalyticsClient
+from correlation import CorrelationProvider
+from fedhist import HistogramProvider
+from provider import AnalyticsProvider
+
+if __name__ == "__main__":
+    d = data.data()
+
+    providers: List[AnalyticsProvider] = [
+        CorrelationProvider(d),
+        HistogramProvider(d),
+    ]
+
+    print(f"starting client with {', '.join(p.name for p in providers) } providers")
```

Comment:
You should be able to use directly: `logging.info` instead (relevant source: https://docs.python.org/3/library/logging.html#logging.basicConfig):

```suggestion
import logging

# ...

    logging.info(f"starting client with {', '.join(p.name for p in providers) } providers")
```